docs/app/global.css

```
@@ -60,16 +60,73 @@ body {
   color: var(--color-fd-primary-foreground);
 }
 
-pre,
+/* Code blocks — match page background in both themes */
+figure.shiki,
+figure.chonkie-codeblock {
+  border-radius: 0.25rem;
+  background-color: var(--color-fd-background) !important;
+  --shiki-light-bg: var(--color-fd-background);
+  --shiki-dark-bg: var(--color-fd-background);
+  box-shadow: none !important;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)